src/libcore/macros.rs

```diff
@@ -228,6 +229,12 @@ macro_rules! try {
         }
     })
 }
+#[cfg(not(stage0))]
+#[macro_export]
+#[stable(feature = "rust1", since = "1.0.0")]
+macro_rules! try {
+    ($expr:expr) => ($expr?)
```

Comment:
to be clear, this change (updating `try!`) was something we wanted to do just to evaluate whether we could stabilize without the carrier trait. Seems clear we cannot -- and so should revert this change, right?